src/core/dfn.js

```diff
@@ -65,6 +59,9 @@ function processAsInternalSlot(title, dfn) {
     showError(msg, name, { hint, elements: [dfn] });
   }
 
+  // Don't export internal slots by default, as they are not supposed to be public.
+  if (!dfn.dataset.hasOwnProperty("export")) dfn.dataset.noexport = "";
```

Comment:
@dontcallmedom, `noexport` is explicit now 🙏

Reply:
thanks :) could you add a test for that case?

Reply:
Yep, see:
https://github.com/w3c/respec/pull/3667/files#diff-67b69965a7e8e8a580e896f9a2101a8fc9490cee82a5357d77c79d66f107b4f2R270-R284

Reply:
Actually, I won't be lazy... I'll add an actual test independent test.